tools/accuracy_checker/openvino/tools/accuracy_checker/launcher/opencv_launcher.py

```diff
@@ -130,6 +133,77 @@ def batch(self):
     def output_blob(self):
         return next(iter(self.output_names))
 
+    def automatic_model_search(self, model_name, model_cfg, weights_cfg, model_type=None):
+        model_type_ext = {
+            'xml': 'xml',
+            'blob': 'blob',
+            'onnx': 'onnx',
+            'caffe': 'prototxt',
+            'paddle': 'pdmodel',
+            'tf': 'pb'
+        }
+        def get_model_by_suffix(model_name, model_dir, suffix):
+            model_list = list(Path(model_dir).glob('{}.{}'.format(model_name, suffix)))
+            if not model_list:
+                model_list = list(Path(model_dir).glob('*.{}'.format(suffix)))
+            if not model_list:
+                model_list = list(Path(model_dir).parent.rglob('*.{}'.format(suffix)))
+            return model_list
+
+        def get_model():
+            model = Path(model_cfg)
+            if not model.is_dir():
+                accepted_suffixes = list(model_type_ext.values())
+                if model.suffix[1:] not in accepted_suffixes:
+                    raise ConfigError('Models with following suffixes are allowed: {}'.format(accepted_suffixes))
+                print_info('Found model {}'.format(model))
+                return model, model.suffix == '.blob'
+            model_list = []
+            if model_type is not None:
+                model_list = get_model_by_suffix(model_name, model, model_type_ext[model_type])
+            else:
+                for ext in model_type_ext.values():
+                    model_list = get_model_by_suffix(model_name, model, ext)
+                    if model_list:
+                        break
+            if not model_list:
+                raise ConfigError('suitable model is not found')
+            if len(model_list) != 1:
+                raise ConfigError('More than one model matched, please specify explicitly')
+            model = model_list[0]
+            print_info('Found model {}'.format(model))
+            return model, model.suffix == '.blob'
+
+        model, is_blob = get_model()
+        if is_blob:
+            return model, None
+        weights = weights_cfg
+        if model.suffix == '.pdmodel':
+            weights = self.get_value_from_config('params')
+        if (weights is None or Path(weights).is_dir()) and model.suffix != '.onnx':
+            weights_dir = weights or model.parent
+            weights_list = []
+            if model.suffix == '.xml':
+                weights = Path(weights_dir) / model.name.replace('xml', 'bin')
+                print(weights)
```

Comment:
if it is debig print, please remove or if it is necessary thing, please replace print to print_info